### PR TITLE
pirate form

### PR DESCRIPTION
**Pull request by longjichen1**
- wrapped most of the form within a div to apply the form background image (wrapped div to exclude the form submit button)
- changed <Button/> to <button/> since <Button/> was difficult to work with and had some interesting gunk
- added svgr webpack dependency (not really important since didn't end up using)
image
image
image

### CODE REVIEW

**Comment by aadibajpai** [comment]
@longjichen1 could we have the same animation from the promo website here as well

**Comment by longjichen1** [comment]
i think zi said he will add those after we get the form up

**Comment by KarolinaGroszewska** [comment]
looks beautiful!

File: models/application.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		},
+		race: {
+			type: Array,
+			required: true,
+		},
+		motivation: {
+			type: Array,
+			required: true,
+		},
+		attendingInPerson: {
```
Comment (KarolinaGroszewska) [inline models/application.ts:54]: is this a field required by MLH or something from previous years we maintained? if the latter, we should remove it as all hackers need to be in-person

**Review by KarolinaGroszewska — COMMENTED** [review]
looks good, just left a q about one of the form fields

**Comment by zineanteoh** [comment]
LGTM

**Pull request merged**